papa/src/main/java/papa/InputTracker.kt

```diff
@@ -36,5 +39,35 @@ interface InputTracker {
 
 class DeliveredInput<T : InputEvent>(
   val event: T,
-  val deliveryUptimeMillis: Long
-)
+  val deliveryUptime: Duration,
```

Comment:
All these years I had no idea we had a Duration class in Kotlin (and also another one in Java!!)